lib/src/features/create_properties/presentation/screens/create_properties.dart

```diff
@@ -1,42 +1,178 @@
+import 'package:chip_list/chip_list.dart';
+
+import 'package:dream_home/src/constants/screen.dart';
+import 'package:dream_home/src/features/create_properties/presentation/views/app_bar.dart';
+import 'package:dream_home/src/features/featured_properties/presentation/widgets/search_properties_field.dart';
+import 'package:dream_home/src/features/user_profile/presentation/widgets/profile_button.dart';
 import 'package:dream_home/src/theme/pellet.dart';
 import 'package:flutter/material.dart';
 
-class CreatePropertyScreen extends StatelessWidget {
+class CreatePropertyScreen extends StatefulWidget {
   const CreatePropertyScreen({super.key});
 
+  @override
+  State<CreatePropertyScreen> createState() => _CreatePropertyScreenState();
+}
+
+class _CreatePropertyScreenState extends State<CreatePropertyScreen> {
+  final numberController = TextEditingController();
+  List<String> userStatus = [
+    'Owner',
+    'Agent',
+    'Builder',
+  ];
+  List<String> isSelling = [
+    'Sell',
+    'Rent/Lease',
+  ];
+  final int selecteduserStatus = 0;
+  final int selectedisSelling = 0;
```

Comment:
I suggest that you rename `selectedisSelling` variable to `selectedISelling` maintaining the integrity of cammelCase.